lib/realtime/tenants/repo/migrations/20240108234812_add_channels_column_for_write_check.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Realtime.Tenants.Migrations.AddChannelsColumnForWriteCheck do
+  @moduledoc false
+
+  use Ecto.Migration
+
+  def change do
+    alter table(:channels, prefix: "realtime") do
+      add :check, :boolean, null: true
```

Comment:
would it be more informative if `check` was set to `false` by default?

Reply:
yeah might be better to keep it with a default value instead of messing around with nil's